tests/openQA/tests.pm

```diff
@@ -4,8 +4,8 @@ use utils;
 
 sub visit_test($needle) {
     assert_and_click 'openqa-all-tests';
-    send_key_until_needlematch $needle, 'down';
```

Comment:
@baierjan you initially introduced the "down" in
* #204

do you agree with the change to "end"?

Reply:
Yeah, go ahead. I guess I introduced the 'down' to make sure the view will scroll down and the needle has chance to match. For long tables 'end' could actually skip some tests. But I guess in our case it just needs to visit one or two test so it shouldn't be an issue, right?